nntrainer/tensor/cl_operations/attention_kernels.h

```diff
@@ -26,6 +26,7 @@ namespace nntrainer {
 // get global cl_context to use in kernels
 static ClContext *attention_cc =
   static_cast<ClContext *>(Engine::Global().getRegisteredContext("gpu"));
+static ClBufferManager &clbuffInstance = ClBufferManager::getInstance();
```

Comment:
Why aren't clbuffInstance.getOutBufferA() and .getOutBufferB() managed as separate variables even though they are used multiple times?